po/pl.po

```diff
@@ -1667,6 +1661,9 @@ msgid ""
 "exits and comments will be deleted. Press Enter to continue or Ctrl+C to "
 "abort"
 msgstr ""
+"OSTRZEŻENIE: Znaleziono kometarz w `%s'/ Plik kolejki jest nadpisywany kiedy "
```

Comment:
Is the slash a typo?

```suggestion
"OSTRZEŻENIE: Znaleziono kometarz w `%s'. Plik kolejki jest nadpisywany kiedy "
```

Reply:
Indeed, what is more it should be kome**n**tarz :facepalm: